ts-client/src/ValorClient.ts

```diff
@@ -0,0 +1,350 @@
+import axios, { AxiosInstance } from 'axios';
+
+type Dataset = {
+  id: number;
+  name: string;
+  metadata: Partial<Record<string, any>>;
+};
+
+type Model = {
+  id: number;
+  name: string;
+  metadata: object;
```

Comment:
metadata's type should be changed to `Partial<Record<string, any>>`

Reply:
updated!